gluten-core/src/main/scala/io/glutenproject/expression/ExpressionConverter.scala

```diff
@@ -115,6 +116,16 @@ object ExpressionConverter extends SQLConfHelper with Logging {
         return replaceScalaUDFWithExpressionTransformer(s, attributeSeq, expressionsMap)
       case _ if HiveUDFTransformer.isHiveUDF(expr) =>
         return HiveUDFTransformer.replaceWithExpressionTransformer(expr, attributeSeq)
+      case i: StaticInvoke =>
+        i.functionName match {
+          case "decode" =>
```

Comment:
I feel it may be a bit unsafe by just matching "decode", which is the method name to be called in the provided static object. Maybe, also check the staticObject of StaticInvoke is `UrlCodec`?

Reply:
hey @PHILO-HE please check if the latest chagne ok for you.